Stateless/DotGraph.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Stateless
+{
+    public partial class StateMachine<TState, TTrigger>
+    {
+        /// <summary>
+        /// A string representation of the state machine in the DOT graph language.
+        /// </summary>
+        /// <returns>A description of all simple source states, triggers and destination states except those defined with PermitDynamic.</returns>

```

Comment:
I have added code such that it will generate individual pseudo-states for each dynamic trigger. An alternative is to have each dynamic trigger point to the same pseudo state, but I think this will  make diagrams slightly less readable.